fe/fe-core/src/main/java/org/apache/doris/nereids/stats/JoinEstimation.java

```diff
@@ -137,7 +137,7 @@ private static Statistics estimateHashJoin(Statistics leftStats, Statistics righ
             outputRowCount = outputRowCount * Math.pow(0.9, unTrustableCondition.size());
         } else {
             outputRowCount = Math.max(leftStats.getRowCount(), rightStats.getRowCount());
-            Optional<Double> ratio = unTrustEqualRatio.stream().max(Double::compareTo);
+            Optional<Double> ratio = unTrustEqualRatio.stream().min(Double::compareTo);
```

Comment:
would better to make a policy for mutiple column ndv estimation based on reversed order, similar with policy in gby?

Reply:
there are some difference. consider two condition: A left join B on A.x=B.x and A.y=B.y
suppose
A left join B on A.x=B.x, output expand A by 3 times,
A left join B on A.y=B.y, output expand A by 5 times.

‘ A left join B on A.x=B.x and A.y=B.y’ output should be less than ‘A left join B on A.x=B.x’